README.md

```diff
@@ -1,17 +1,22 @@
 # Data Science Salaries Dashboard
 ## About
 
-This project aims to build a [dashboard](https://datasci-salaries-py.herokuapp.com/) to visualize data science salaries across the world. Further details can be found in our proposal [here](https://github.com/UBC-MDS/datasci_salaries_py/blob/main/proposal.md).
+This project aims to build a [dashboard](https://datasci-salaries-py.herokuapp.com/) to visualize data science salaries across the world. Further details can be found in our proposal [here](https://github.com/UBC-MDS/datasci_salaries_py/blob/main/doc/proposal.md).
```

Comment:
Sorry, I don't know if I'm being too picky but I think it would be better to explicitly mention the link to our dashboard

```suggestion
This project aims to build a dashboard to visualize data science salaries across the world. Further details can be found in our proposal [here](https://github.com/UBC-MDS/datasci_salaries_py/blob/main/doc/proposal.md).

Link to dashboard: https://datasci-salaries-py.herokuapp.com/
```

Reply:
I updated it!

Reply:
Thanks!

Reply:
Thank you for your suggestions!